src/test/java/com/commercetools/emailprocessor/model/StatisticsTest.java

```diff
@@ -0,0 +1,84 @@
+package com.commercetools.emailprocessor.model;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class StatisticsTest {
```

Comment:
You don't need to test getters and setters.

Reply:
c154e43